hugegraph-llm/src/hugegraph_llm/operators/llm_op/info_extract.py

```diff
@@ -165,110 +65,62 @@ def split_string_to_fit_token_space(
     return combined_chunks
 
 
-def get_spo_from_result(result):
-    res = []
-    for row in result:
-        row = row.replace("\\n", "").replace("\\", "")
-        pattern = r'\("(.*?)", "(.*?)", "(.*?)"\)'
-        res += re.findall(pattern, row)
-    return res
+def extract_by_regex(text, triples):
```

Comment:
```suggestion
def extract_triples_by_regex(text, triples):
```

`v-n-adv` phrase maybe better?